### PR TITLE
Update lint, web gen for OFL and emoji font location changes

### PR DESCRIPTION
**Pull request by dougfelt**

### CODE REVIEW

File: nototools/generate_website_2_data.py [hunk 1]
```diff
@@ -246,8 +246,7 @@ def get_family_id_to_regions(family_id_to_lang_scr_to_sample_key):
 
   family_id_to_regions = collections.defaultdict(set)
   warnings = set()
-  for (family_id, lang_scr_to_sample_key in
-       family_id_to_lang_scr_to_sample_key.iteritems()):
+  for family_id, lang_scr_to_sample_key in family_id_to_lang_scr_to_sample_key.iteritems():
```
Comment (jamesgk) [inline nototools/generate_website_2_data.py]: Accidental change? The line is now over 80 characters.

**Comment by jamesgk** [comment]
LGTM

File: nototools/generate_website_2_data.py (hunk 1 above)
Comment (dougfelt) [inline nototools/generate_website_2_data.py]: When I tried to run this with a wrapped line, it failed.  Thought I had run it with the wrapped line, but maybe I hadn't.  Anyway, I had to unwrap it to make it work.

Should the wrapped version work?

Comment (roozbehp) [inline nototools/generate_website_2_data.py]: Try unpacking the tuple inside the loop?

Comment (dougfelt) [inline nototools/generate_website_2_data.py]: That is frighteningly obvious, isn't it... Not my day.

On Tue, Sep 29, 2015 at 2:19 PM, Roozbeh Pournader <notifications@github.com

**Pull request merged**